platform-core-module/src/main/java/com/softicar/platform/core/module/ajax/listener/AjaxContextListener.java

```diff
@@ -47,5 +48,6 @@ public void contextDestroyed(ServletContextEvent sce) {
 		new IioRegistryManager().deregisterProviders();
 		servletContext = null;
 		Log.finfo("CONTEXT DESTROYED: %s", sce.getServletContext().getContextPath());
+		CurrentSingletonSet.reset();
```

Comment:
Seems like we want to guarantee this call to happen. If so, we should probably wrap everything above in a `try` block, and call `CurrentSingletonSet.reset();` in a corresponding `finally` block.

Reply:
added `try...finally`